tree/readspeed/src/ReadSpeedCLI.cxx

```diff
@@ -129,7 +129,18 @@ const auto fullUsageText =
    " first read. If this is realistic of how your analysis will run - then there is no concern. However, if\n"
    " you expect to only read files once in a while - and as such the files are unlikely to be in the cache -\n"
    " consider clearing the cache before running rootreadspeed.\n"
-   "  (On Linux this can be done by running 'echo 3 > /proc/sys/vm/drop_caches' as a superuser)\n";
+   " On Linux this can be done by running 'echo 3 > /proc/sys/vm/drop_caches' as a superuser\n"
+   " or a specific file can be dropped from the cache with"
+   " `dd of=<FILENAME> oflag=nocache conv=notrunc,fdatasync count=0 > /dev/null 2>&1`.\n"
+   "\n"
+   "\n"
+   "Known overhead of TTreeReader, RDataFrame:\n"
+   "`rootreadspeed` is designed to read all data present in the specified branches, trees and files at the highest\n"
+   "possible speed. When an analysis application is bound by I/O, higher-level interfaces built on top of TTree\n"
+   "such as TTreeReader and RDataFrame are known to add a significant runtime overhead with respect to the runtimes\n"
```

Comment:
If the analysis is I/O bound then extra CPU cycles shouldn't matter? Not sure I understand this...

Reply:
Here it's I/O as in "the ROOT I/O machinery" (which `rootreadspeed` can use more efficiently than `TTreeReader` does).

I'll change it to "When the bottleneck is not in the analysis logic..."